packages/cli/commands/project/listBuilds.js

```diff
@@ -0,0 +1,173 @@
+const { getCwd } = require('@hubspot/cli-lib/path');
+const { validateAccount } = require('../../lib/validation');
+const { link } = require('../../lib/links');
+const { getProjectConfig, getProjectDetailUrl } = require('../../lib/projects');
+const moment = require('moment');
+const { prompt } = require('inquirer');
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validateAccount(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'list-builds [path]';
+exports.describe = false;
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const { path: projectPath, limit } = options;
+  const accountId = getAccountId(options);
+
+  trackCommandUsage('project-list-builds', { projectPath }, accountId);
+
+  const cwd = projectPath ? path.resolve(getCwd(), projectPath) : getCwd();
+  const projectConfig = await getProjectConfig(cwd);
+
+  logger.debug(`Fetching builds for project at path: ${projectPath}`);
+
+  const fetchAndDisplayBuilds = async (project, options) => {
+    const { results, paging } = await fetchProjectBuilds(
+      accountId,
+      project.name,
+      options
+    );
+    const currentDeploy = project.deployedBuildId;
+    if (options && options.after) {
+      logger.log(
+        `Showing the next ${results.length} builds for ${project.name}`
+      );
+    } else {
+      logger.log(
+        `Showing the ${results.length} most recent builds for ${project.name}. ` +
+          link(
+            'View all builds in project details.',
+            getProjectDetailUrl(project.name, accountId)
+          )
+      );
+    }
+
+    if (results.length === 0) {
+      logger.log('No builds found.');
+    } else {
+      const builds = results.map(build => {
+        const isCurrentlyDeployed = build.buildId === currentDeploy;
+
+        return [
+          isCurrentlyDeployed
+            ? `#${build.buildId} [deployed]`
+            : `#${build.buildId}`,
+          build.status,
+          moment(build.finishedAt).format('MM/DD/YY, hh:mm:ssa'),
+          Math.round(
+            moment
+              .duration(moment(build.finishedAt).diff(moment(build.enqueuedAt)))
+              .asSeconds()
+          ) + 's',
+          build.subbuildStatuses
+            .filter(subbuild => subbuild.status === 'FAILURE')
+            .map(subbuild => `${subbuild.buildName} failed`)
+            .join(', '),
+        ];
+      });
+      builds.unshift(
+        getTableHeader([
+          'Build ID',
+          'Status',
+          'Completed',
+          'Duration',
+          'Details',
+        ])
+      );
+      logger.log(
+        getTableContents(builds, {
+          columnDefault: {
+            paddingLeft: 3,
+          },
+        })
+      );
+    }
+    if (paging && paging.next) {
```

Comment:
Sorry if this is already a known issue, but it looks like `paging.next` exists even if I only have one build. So this prompt is showing when it shouldn't.

Reply:
Yep! This endpoint always returns an empty "last page" and I flagged with the BE folks and they have a fix in progress